### PR TITLE
Add visitor and default axis to CumSum for opset3

### PR DESCRIPTION
**Pull request by adstraw**

### CODE REVIEW

File: src/ngraph/op/cum_sum.cpp [hunk 1]
```diff
@@ -35,6 +36,20 @@ op::v0::CumSum::CumSum(const Output<Node>& arg,
     constructor_validate_and_infer_types();
 }
 
+op::v0::CumSum::CumSum(const Output<Node>& arg, const bool exclusive, const bool reverse)
+    : Op({arg, op::Constant::create(element::i32, Shape{}, {0})})
+    , m_exclusive(exclusive)
```
Comment (diyessi) [inline src/ngraph/op/cum_sum.cpp:41]: Needs to be a new op version.

Comment (lazarevevgeny) [reply src/ngraph/op/cum_sum.cpp:41]: I always vote for a version change for each change in the operation unless it an obvious bug fix. But for this particular case we may keep the version.

**Comment by raramer01** [comment]
TESTNOW

**Comment by raramer01** [comment]
TESTNOW_DLDT_E2E

**Comment by raramer01** [comment]
TESTNOW

**Comment by adstraw** [comment]
TESTNOW

**Comment by raramer01** [comment]
TESTNOW_DLDT_E2E

**Pull request merged**